
package co.com.colfondos.tech.ws.cliente.funcionario.service_funcionario;

import javax.xml.ws.WebFault;


/**
 * This class was generated by the JAX-WS RI.
 * JAX-WS RI 2.2.9-b130926.1035
 * Generated source version: 2.2
 * 
 */
@WebFault(name = "InternalException", targetNamespace = "http://funcionario.business.colfondos.com.co/")
public class InternalException
    extends Exception
{

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private InternalFault faultInfo;

    /**
     * 
     * @param faultInfo
     * @param message
     */
    public InternalException(String message, InternalFault faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param faultInfo
     * @param cause
     * @param message
     */
    public InternalException(String message, InternalFault faultInfo, Throwable cause) {
        super(message, cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *     returns fault bean: co.com.colfondos.tech.ws.cliente.funcionario.service_funcionario.InternalFault
     */
    public InternalFault getFaultInfo() {
        return faultInfo;
    }

}
